### DISCUSSION TITLE
Arch linux Wayland: fatal display error: Protocol error

### DISCUSSION THREAD

**Discussion by Strykar** [kovidgoyal/kitty#7239] [Q&A] [answered]
```
$ /usr/bin/kitty
[079 01:03:14.694845] Ignoring invalid font feature: +script
[079 01:03:14.694926] Ignoring unknown config key: share_connections
[destroyed object]: error 7: failed to import supplied dmabufs: Unsupported buffer format 875713345
[079 01:03:14.787123] [glfw error 65544]: Wayland: fatal display error: Protocol error
```
This issue started today on Arch linux -
```
$ kitty --version
kitty 0.33.0 created by Kovid Goyal 
```

**Comment by Strykar** [comment]
```
$ kitty --debug-rendering --debug-gl -c /usr/share/doc/kitty/kitty.conf
Creating window at size: 1816x640 and scale 1
GL version string: '3.1.0 NVIDIA 550.54.14' Detected version: 3.1
CSD: old.size: 0x0 new.size: 1816x640 needs_update: 1 size_changed: 1 buffer_destroyed: 0
Created decoration buffers at scale: 1 vertical_height: 676 horizontal_width: 1840
top-level configure event: size: 0x0 states:
CSD: old.size: 1816x640 new.size: 1816x640 needs_update: 0 size_changed: 0 buffer_destroyed: 0
final window content size: 1816x640 resized: 0
Setting window geometry in configure event: x=0 y=-24 1816x664
CSD: old.size: 1816x640 new.size: 1816x640 needs_update: 1 size_changed: 0 buffer_destroyed: 1
Created decoration buffers at scale: 1 vertical_height: 676 horizontal_width: 1840
CSD: old.size: 1816x640 new.size: 1816x640 needs_update: 0 size_changed: 0 buffer_destroyed: 0
Waiting for swap to commit: swap has happened
[destroyed object]: error 7: failed to import supplied dmabufs: Unsupported buffer format 875713345
Calling wl_pointer_set_cursor in setCursorImage with surface: 0x5ddc465372e0
[079 06:34:30.997446] [glfw error 65544]: Wayland: fatal display error: Protocol error
```

**Comment by kovidgoyal** [comment, marked as answer]
dup of #7174

On Mon, Mar 18, 2024 at 02:12:43PM -0700, Avinash H. Duduskar wrote:
> ```
> $ /usr/bin/kitty
> [079 01:03:14.694845] Ignoring invalid font feature: +script
> [079 01:03:14.694926] Ignoring unknown config key: share_connections
> [destroyed object]: error 7: failed to import supplied dmabufs: Unsupported buffer format 875713345
> [079 01:03:14.787123] [glfw error 65544]: Wayland: fatal display error: Protocol error
> ```
> This issue started today on Arch linux -
> ```
> $ kitty --version
> kitty 0.33.0 created by Kovid Goyal 
> ```
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/7239
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________

**Comment by Strykar** [comment]
Thanks, currently worked around by setting `linux_display_server x11` in [kitty.conf](https://www.reddit.com/r/KittyTerminal/comments/1bhzneq/comment/kvj38ve/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button).
